server/views/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>Csv</title>
+      <meta charset="utf-8" />
+      <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class="container" onclick="closeCover()"></div>
+    <div class="head">
+        <form name="uploadFile" class="uploadFile">
+          <input type="file" id="getFile" name="myFile" onchange="upload(this.files[0])" class="myFileInput">
+        </form>
+
+        <button class="newFile" onclick="upload('newfile')">New File</button>
+        <button class="newFile" onclick="download()">Save File</button>
+        <button class="newFile" onclick="showModal()">Open Files</button>
+        <input class="cellInputForm" onkeydown = "if (event.keyCode == 13) writeCell()">
```

Comment:
move if to function and pass event to it

Reply:
done